index.js

```diff
@@ -300,30 +310,45 @@ function logError(context, error){
             page++;
             const json = JSON.parse(response.body);
             nbResults = json.hotspots.length;
-            data.issues = data.issues.concat(json.hotspots.map(hotspot => {
-              hSeverity = hotspotSeverities[hotspot.vulnerabilityProbability];
+            data.hotspotKeys = json.hotspots.map(hotspot => hotspot.key);
```

Comment:
this should append to hotspotKeys instead of overwriting for each page response.

Reply:
https://github.com/soprasteria/sonar-report/pull/86